pkg/console/operator/sync_v400.go

```diff
@@ -159,17 +163,17 @@ func (co *consoleOperator) sync_v400(updatedOperatorConfig *operatorv1.Console,
 	if consoleURL == "" {
 		err := customerrors.NewSyncError("waiting on route host")
 		klog.Errorf("%q: %v", "route", err)
-		return err
+		conditionHandler.FlushAndReturn(err)
```

Comment:
Is it your intent to ignore the error returned by FlushAndReturn?

Reply:
err...